src/aerie_cli/__main__.py

```diff
@@ -23,19 +25,40 @@
 app.add_typer(scheduling.app, name="scheduling")
 app.add_typer(metadata.app, name="metadata")
 
+
+def print_version(print_version: bool):
+    if print_version:
+        typer.echo(__version__)
+        raise typer.Exit()
+
+
 @app.callback()
-def setupGlobalCommandContext(
-        hasura_admin_secret = typer.Option(default="",
-                                           help="Hasura admin secret that will be put in the header of graphql requests")
-    ):
+def app_callback(
+    version: Optional[bool] = typer.Option(
+        None,
```

Comment:
Why not provide `-v` as an option?

Reply:
Ope good catch, I will add that back in. I had it at one point, but this is, like, rev 5 of how to implement this and I lost it.